packages/html/ds/src/list/list.schema.ts

```diff
@@ -0,0 +1,16 @@
+import * as zod from 'zod';
+import { linkSchema } from '../link/link.schema';
+
+export enum TypeEnum {
+  Normal = 'normal',
+  Bullet = 'bullet',
+  Number = 'number',
+}
+
+const linkPropsSchema = zod.object({
+  items: zod.array(zod.union([zod.string(), linkSchema])),
+  spaced: zod.boolean().optional(),
+  type: zod.nativeEnum(TypeEnum).optional(),
+});
```

Comment:
I think you'll get an error here when you do `pnpm build` in `packages/html/ds` because there is a description required for each of the properties

Reply:
where could I find the description required?

I ran the  pnpm build and I got all succeeded, should I go inside of `packages/html/ds` and run pnpm build?

Reply:
yes, you have to go into `packages/html/ds` and there run `pnpm build`. For some reason the errors don't show when you do pnpm build in the root